src/main/java/com/softserve/teamproject/controller/UserProfileController.java

```diff
@@ -36,11 +36,11 @@ public User getUserProfile(Principal principal) {
   }
 
   /**
-   * Get groups from user location
+   * Get groups from the authenticated user location
    * @param principal authorized user
    * @return groups in user location
    */
-  @RequestMapping(value = "/user/location/groups", method = RequestMethod.GET)
+  @RequestMapping(value = "/groups/locations/my", method = RequestMethod.GET)
```

Comment:
he didn't like when we have unnecessary  /<USER>/location/groups.
Restful way is  URI way.   We dont have any identifier in /user/location/groups.
But my option has one /locations/{MY}/groups